### PR TITLE
Adjust login and sign up

### PR DESCRIPTION
**Pull request by Oozaru71**
Made quite a few changes but i left a summary of the most noticeable ones but now everything should be as expected,

### CODE REVIEW

File: Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,91 +1,1358 @@
...
+<body class="">
+  <section class="vbox">
+    <header class="bg-white-only header header-md navbar navbar-fixed-top-xs">
+      <div class="navbar-header aside bg-info nav-xs">
+        <a class="btn btn-link visible-xs" data-toggle="class:nav-off-screen,open" data-target="#nav,html">
+          <i class="icon-list"></i>
+        </a>
+        <a href="index.html" class="navbar-brand text-lt">
+          <i class="icon-earphones"></i>
+          <img src="/css2/images/logo.png" alt="." class="hide">
```
Comment (Alejandro-HUB) [inline Views/Home/Index.cshtml:38]: I mentioned them on the video but I’ll write it here too.

1) The logo on the top left
2) Logout button
3) Positioning of the music bar
4) Positioning of the picture on the left panel

**Review by Alejandro-HUB — CHANGES_REQUESTED** [review]

File: Views/Home/Index.cshtml [hunk 2]
```diff
@@ -118,7 +121,7 @@
               </li>
               <li class="divider"></li>
               <li>
-                 <button type="button" class="logoutbtn" href="#" onclick="signOut()" id="logout">LOG OUT</button>
+                <a href="modal.lockme.html" data-toggle="ajaxModal" >Logout</a>
```
Comment (Alejandro-HUB) [inline Views/Home/Index.cshtml]: To add the logic for the logout button you need to go inside the ajax modal that gets triggered once you click your name. You will find the logout button inside modal.lockme.html.

Also, you do not need to create another pull request when you push your changes the pull request gets automatically updated. Which is why I am denying your other pull request.

Finally, the logo is a css issue all you need to do is look at which is one has css that messes up the other. If you think its the user picture then look into the user picture's css and make it smaller or play around with it.

**Review by Alejandro-HUB — CHANGES_REQUESTED** [review]

**Comment by Oozaru71** [comment]
I have fixed both problems, log out and avatar pic size.

**Review by Alejandro-HUB — APPROVED** [review]
Great job, it's looking good. I will assign a new task to you tomorrow 👍

**Pull request merged**